dataedit/templates/dataedit/dataview.html

```diff
@@ -5,7 +5,8 @@
 
 
 {% block site-header %}
-  <h2 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>.{{ table }}</h2>
+  <h2 class="site-header">{{ title }}</h2>
```

Comment:
There should be a fallback, if there is no title set in the metadata string. I would recommend the following structure

{% if title %}
  <h3 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>: {{ title }}</h3>
{% else %}
  <h3 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>.{{ table }}</h3>
{% endif %}

Reply:
Review Comment addressed.